td.vue/src/i18n/ar.js

```diff
@@ -47,7 +47,12 @@ const ara = {
         }
     },
     demo: {
-        select: 'حدد نموذج تهديد تجريبي من القائمة أدناه'
+        select: 'حدد نموذج تهديد تجريبي من القائمة أدناه',
+        "New Threat Model": "نموذج تهديد جديد",
```

Comment:
the names are not ideal, such as "New Treat Model", and should be renamed to something more in keeping with the rest of the attribute naming